nix/nixpkgs.nix

```diff
@@ -8,6 +8,20 @@ let
 
   # package overrides
   overrides = _: pkgs: rec {
+    # nixpkgs ships with a very old version of openjdk on darwin
```

Comment:
I don’t think this is actually required but the fact that we are on a pretty old JDK doesn’t inspire confidence and I’d rather not waste time debugging issues that have already been fixed.